src/libs/TransactionUtils.ts

```diff
@@ -360,6 +361,13 @@ function getValidWaypoints(waypoints: WaypointCollection, reArrangeIndexes = fal
     }, {});
 }
 
+/**
+ * Returns the most recent transactions in an object
+ */
+function getRecentTransactions(transactions: Record<string, string>, size = 2): unknown[] {
```

Comment:
```suggestion
function getRecentTransactions(transactions: Record<string, string>, size = 2): string[] {
```

Also we should try not to use `unknown` wherever possible. This just returns a string array right?

Reply:
It does, but `_.sortBy` returns unknown[], so the linter complains if I return string[] here.

Reply:
Even more reason not to use lodash 😄